### ISSUE TITLE
undefined method `negotiate' for Net::NTLM:Module

### ISSUE THREAD

**Issue by alexphys** [macks/ruby-ntlm#6]
Hi ,
I've included the
```ruby
require 'ntlm/smtp'
```
inside `config/environment.rb`.
My settings inside `config/application.rb` are as following;
```ruby
   config.active_support.deprecation = :notify
    config.action_mailer.delivery_method = :smtp
    config.action_mailer.smtp_settings = {
      :address => "*ADDRESS*",
      :domain => '*DOMAIN*',
      :user_name => '*USERNAME*',
      :password => '*PASSWORD*',
      :port => 25,
      :authentication => :ntlm
    }
```
and I get the ` undefined method 'negotiate' for Net::NTLM:Module`
I can see that this was supposed to be [fixed](https://github.com/macks/ruby-ntlm/commit/2a4e344c4b38efc11cb7b5568ddc0064550a6c76) but it still occurs.

Any suggestions?

**Comment by Maarcoon** [comment]
Same here

**Comment by Canutza** [comment]
Hello,

You managed to resolve this? if so, how ?

**Comment by Maarcoon** [comment]
I pointed the source to github when I add the gem to my Gemfile

The problem is that there is no updates on rubygems for this gem since 2011 and the commit that solves this was added in 2013

gem 'ruby-ntlm', :git => 'https://github.com/macks/ruby-ntlm'

**Comment by macks** [comment]
Fixed in 0.0.3
